repo/blob/s3/s3_options.go

```diff
@@ -1,5 +1,9 @@
 package s3
 
+import (
+	"time"
+)
```

Comment:
To match with the other PR's change? 
```suggestion
import "time"
```
https://github.com/kastenhq/kopia/pull/230